site/_scss/layouts/devsite-theme.scss

```diff
@@ -1,9 +1,877 @@
+            }
+          }
+        }
+      }
+    }
+
+    @media (max-width: 591px) {
+      --devsite-card-border-radius: 0;
+      --devsite-landing-group-padding: 0;
+    }
+  }
+
+  .dcc-case-studies {
+    --devsite-landing-row-group-background: #{$BG_PRIMARY};
+    --devsite-landing-row-group-border-radius: #{$CARD_BORDER_RADIUS};
+    --devsite-landing-row-group-max-width: 90rem;
+    --devsite-landing-row-group-padding: 2rem 1rem 1rem;
+    --devsite-item-gap: 0;
+    
+    [description-position=bottom] {
+      --devsite-item-gap: 0;
+      --devsite-item-media-margin: 0 0 0 1rem;
+      --devsite-buttons-margin: 0.5rem 0 0 -0.75rem;
+      padding: 2.125rem;
+
+      &:first-child {
+        --devsite-item-image-width: 2.5rem;
+        --devsite-item-media-container-width: 2.5rem;
+        --devsite-item-text-align: left;  
+        --devsite-h2-font: 700 2.5rem/3rem #{$TEXT_PRIMARY_FONT_FAMILY};
+
+        padding-right: 0;
+        padding-top: 1rem;
+
+        figure {
+          margin-bottom: 0.25rem;
+        }
+
+        @media (max-width: 1290px) {
+          grid-column: 1 / span 3;
+        }
+
+        @media (max-width: 840px) {
+          padding: 1.125rem 0;
+        }
+      }
+
+      &:last-child {
+        --devsite-h2-font: 500 2rem/2.375rem #{$TEXT_PRIMARY_FONT_FAMILY};
+        padding-left: 0;
+        padding-top:4rem;
+
+        @media (max-width: 1290px) {
+          grid-column: 2 / span 2;
+          padding-top: 0;
+
+          --devsite-h2-font: 500 1.5rem #{$TEXT_PRIMARY_FONT_FAMILY};
+        }
+
+        @media (max-width: 840px) {
+          padding: 0;
+          grid-column: 1/span2;
+        }
+      }
+
+      @media (max-width: 840px) {
+        padding: 1.125rem;
+        grid-column: 1 / span 3;
+      }
+    }
+
+    [description-position=top] {
+      --devsite-description-content-font: 0 #{$TEXT_PRIMARY_FONT_FAMILY};
+
+      display: block;
+      padding: 5rem 1rem 0rem;
+
+      figure {
+        padding: 0 0 0 1rem;
+      }
+
+      @media (max-width: 1290px) {
+        padding-top: 2.5rem;
+
+        span {
+          top: 0;
+        }
+      }
+
+      @media (max-width: 840px) {
+        padding: 2.5rem 1.125rem 0;
+        grid-column: 1 / span 3;
+
+        > div:last-child {
+          padding: 0;
+        }
+
+        figure {
+          margin-bottom: 0;
+          padding: 0 0 1rem 0;
+        }
+      }
+    }
+
+    @media (max-width: 591px) {
+      --devsite-landing-group-margin: 0 0 2rem 0;
+      --devsite-landing-group-padding: 0;
+      padding: 0;
+    }
+
+  }
+
+  .dcc-chrome-releases {
+    --devsite-landing-row-header-max-width: 17.5625rem;
+    --devsite-landing-row-header-padding: 0.5rem 0;
+
+    position: relative;
+
+    .dcc-blog-card {
+      max-width: 24.75rem;
+      padding: 0;
+      
+      figure{
+        margin-bottom: 0;
+        padding: 0;
+      }
+
+      img{
+        height: 12.5rem;
+        object-fit: cover;
+      }
+    }
+
```

Comment:
Whitespace.

Reply:
@simgobb, @mamieorine: that's a double new-line which can be condensed to one.

Reply:
We removed this double new line. Please see the [latest file here](https://github.com/GoogleChrome/developer.chrome.com/blob/4956d82e2973630b36fd268f5e720413ce6baa7c/site/_scss/layouts/devsite-theme.scss.). Thank you 😊